src/core/core/rpc_message.cpp

```diff
@@ -214,6 +214,18 @@ message_ex::message_ex()
 
 message_ex::~message_ex()
 {
+    // when receiving a request, the lifetime of message_header object is managed by
+    // vector "buffers" (see@create_receive_message function).
+    // if you call copy_and_prepare_send() function, then a new request will be created,
+    // but new request shares the same header with the old_request.
+    // so if old_request releases header,
+    // then new request's header will be invalid.
+    //
+    // so we don't release_buffer_header().
+    // however, this won't lead to any memory leak problems as long as the header is a POD type
```

Comment:
这里说"the header is a POD type"

Reply:
还是写清楚点吧：however, this won't lead to any memory leak problems as long as the header can be thought of as POD type (because the from_address must be IPv4 address), refer to the ATTENTION of message_header.

Reply:
后来这个地方改过了，你看看新的注释，应该也能说清楚了